Pirate-Wires/pirate-wires
# Bugfix/customer io race condition

QA instructions:

1. make sure you are signed out
2. via any newsletterCallout i.e. an email input with a "subscribe" button, enter an email for a user you know does not exist
3. observe, toast confirms that subscription prefs were saved
4. repeat 2 with a user you know exists
5. observe, toast confirms that subscription prefs were saved
6. confirm customer records exist in customer.io
